molr-mole-core/src/main/java/io/molr/mole/core/runnable/demo/conf/DemoRunnableLeafsConfiguration.java

```diff
@@ -177,8 +178,8 @@ public RunnableLeafsMission foreachMission(){
     			 * map called on foreach will map/transform each item into an object created by the given factory
     			 */
     			root("foreachDemo").foreach(someDevices).parallel().map(DeviceDriver::new).branch("work on device {} branch", Placeholders.LATEST_FOREACH_ITEM_PLACEHOLDER).as((doWithDeviceBranch, devicePlaceholder)-> {
-    				doWithDeviceBranch.leaf("SwitchOn ").runFor(device->{device.switchOn();});
-    				doWithDeviceBranch.leaf("Pause").run(()->Thread.sleep(1000));
+    				doWithDeviceBranch.leaf("SwitchOn ").perDefault(BlockAttribute.IGNORE).runFor(device->{device.switchOn();});
+    				doWithDeviceBranch.leaf("Pause").perDefault(BREAK).run(()->Thread.sleep(1000));
```

Comment:
Here it is: We started with block attributes already ;-)
So to generalize, we just should keep this in a (multi-) map and not do copy paste code ....  This way it is again a general API, while other moles might not provide the same attributes....